base/db/R/dbfiles.R

```diff
@@ -120,9 +121,17 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
         "'" , parent, ";"
       ),
       con = con
-    )[['id']]
+    )$id
   }
-
+  
+  if (length(inputid) > 1) {
+    PEcAn.logger::logger.warn(paste0("Multiple input files found matching parameters format_id = ", formatid, 
+                                     ", startdate = ", startdate, ", enddate = ", enddate, ", parent = ", parent, ".  Selecting the", 
+                                     " last input file.  This is normal for when an entire ensemble is inserted iteravely, but ", 
+                                     " is likely an error otherwise."))
+    inputid = inputid[length(inputid)]
```

Comment:
This gave me a really weird bug.  

The function `dbfile.input.insert` is shielded from handling entire ensembles at once; instead, it is called iteratively in `convert.input` to insert files one at time.  At this point in the function, `inputid` is used only to properly set the `containter_id` when creating the associated `dbfile`.

The query used to grab `inputid` from the database isn't specific enough to filter out IDs based on ensemble.  Instead, it returns a vector of `inputid`s which all match the query.  The magic of R selects the first inputid in the list to use as the container ID for the dbfile if the length of inputid is more than one.  As a result, the first ensemble member inserts properly, but the 2nd - 21st ensemble members fail to insert because their dbfile is associated with the first ensemble's input id.  The query used to find the `inputid`s returns in numerically ascending order, which is also the order that the ensembles have been inserted.  Instead of the first element of the vector, which will always be for the first ensemble, we want the last one, which is the id for the most recently created input file - the input file for the ensemble we are inserting.